src/saml2/cert.py

```diff
@@ -290,7 +290,7 @@ def verify_chain(self, cert_chain_str_list, cert_str):
 
     def certificate_not_valid_yet(self, cert):
         starts_to_be_valid = dateutil.parser.parse(cert.get_notBefore())
-        now = pytz.UTC.localize(datetime.datetime.utcnow())
+        now = datetime.datetime.now(dateutil.tz.tzutc())
```

Comment:
If you ever want to pin `python-dateutil>=2.7.0`, you can replace this with `dateutil.tz.UTC`.

Reply:
I would probably use the already created instance than calling for new one (even though it is a singleton).
```py
now = datetime.datetime.now(dateutil.tz.UTC)
```

Reply:
It's up to you, I went with `tz.tzutc()` because for most libraries the additional benefit of using the pre-created instance is minimal compared to the cost of pinning a dependency (which can lead to conflicts with some applications that pin to a different version). It's a bit slower to call `tz.tzutc()`, but I'm guessing that this function is never called in a tight loop, so 300ns doesn't make much difference either way:

```python
In [3]: %timeit datetime.now(tz.UTC)
1.3 µs ± 7.18 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [4]: %timeit datetime.now(tz.tzutc())
1.65 µs ± 5.65 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```